app/scripts/components/common/map/style-generators/raster-timeseries.tsx

```diff
@@ -272,14 +272,35 @@ export function RasterTimeseries(props: RasterTimeseriesProps) {
         LOG && console.log('Payload', payload);
         LOG && console.groupEnd();
         /* eslint-enable no-console */
-
-        const responseData = await requestQuickCache<any>({
-          url: `${tileApiEndpointToUse}/mosaic/register`,
-          payload,
-          controller
-        });
-
-        setMosaicUrl(responseData.links[1].href);
+        
+        let responseData;
+        let fallbackToOld = false;
+        try {
+          responseData = await requestQuickCache<any>({
+            url: `${tileApiEndpointToUse}/searches/register`,
+            payload,
+            controller
+          });
+        } catch (e) {
+          // @NOTE: conditional logic TO BE REMOVED once new BE endpoints have moved to prod... Fallback on old request url if new endpoints error with nonexistance... 
+          if (e.request) {
+            // The request was made but no response was received
+            responseData = await requestQuickCache<any>({
+              url: `${tileApiEndpointToUse}/mosaic/register`, // @NOTE: This will fail anyways with "staging-raster.delta-backend.com" because its already deprecated...
+              payload,
+              controller
+            });
+            fallbackToOld = true;
+          }
+        }
```

Comment:
Should we have an else block to throw an error in case the error is not from the new/old API endpoint problem?

Reply:
I added an else block with a `LOG` following the patterns in this file, do you think this is sufficient enough?